### ISSUE TITLE
Why CHOPPER_DEFAULT_12v instead of CHOPPER_DEFAULT_24V?

### ISSUE THREAD

**Issue by joirma** [bigtreetech/BIGTREETECH-SKR-E3-DIP-V1.0#35]
Why is it configured with the parameter CHOPPER_DEFAULT_12v instead of CHOPPER_DEFAULT_24V?
To ender 3 the correct property would not be CHOPPER_DEFAULT_24V?

**Comment by Libelulas** [comment]
Anyone can say anything? is a interesting question, becouse all of the youtube tutorials don't say anything about this... have you tested the 24V option? I Will to do it...

**Comment by MohanMaker** [comment]
Yes, it should have been set to CHOPPER_DEFAULT_24V for the Ender 3, and this is fixed in the new firmware files on this repository. This setting affects the StealthChop mode when using TMC drivers.

**Comment by oldman4U** [comment]
Hello joirma.

Could you please close this ticket.

Thank you

**Comment by oldman4U** [comment]
@bigtreetech Could you please close this unneeded ticket. Thank you
